src/Transpiler.ts

```diff
@@ -685,7 +685,8 @@ export abstract class LuaTranspiler {
         if (node.expression) {
             // If parent function is a TupleReturn function
             // and return expression is an array literal, leave out brackets.
-            const declaration = tsHelper.findFirstNodeAbove(node, ts.isFunctionDeclaration);
+            const declaration = tsHelper.findFirstNodeAbove(node, ts.isFunctionDeclaration)
```

Comment:
I would prefer if we only did the search once, I think `tsHelper.findFirstNodeAbove(node, n => ts.isFunctionDeclarations(n) || ts.isMethodDeclaration(n))` should be fine.

Reply:
Thanks and good idea, added another commit to do this search once

Reply:
Looks good!